modules/ROOT/pages/to-share-api-asset-to-portal.adoc

```diff
@@ -4,6 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
+An organization's Exchange administrators control whether each API asset created in the organization is shared in the public portal. Set which users have the Exchange administrator role in the Access Management > Roles menu, and find more information in xref:to-set-permissions.adoc[Set User Access Permissions].
+
+== Add an Asset to a Public Portal
+
 . Click an API asset to show its details.
 . Click Share and click *Add to the public portal*.
 . (Optional) Click Share Internally if you decide to not put the asset in the public portal and instead return
```

Comment:
Instead of beginning optional steps this way, consider starting them with "If you want to do this,....
GLOBAL

Reply:
Click *Share Internally*
GLOBALLY format labels of items that your users is clicking or selecting.

Reply:
End sentence at "public portal."

QUESTION: Are assets by default shared internally? Do we need to clarify this, if so, or have separate steps for adding for public use vs. adding for internal use?